server/src/controllers/logout.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response } from 'express';
+
+const logout = (req:Request, res:Response) => {
+  res.clearCookie('token').clearCookie('token').json({ redirect: '/' });
```

Comment:
why do you clear the cookie twice ?!
I think it's not necessary 

and i prefere to return a message in the response
```suggestion
  res.clearCookie('token').json({ status: 200, msg: 'logged out successfully !' });
```